doc/classes/RenderSceneBuffersRD.xml

```diff
@@ -61,15 +61,15 @@
 			<param index="1" name="msaa" type="bool" default="false" />
 			<description>
 				Returns the specified layer from the color texture we are rendering 3D content to.
```

Comment:
I'd like to change this "we are rendering" in the meantime, but I am not sure what is the most appropriate here.